app/src/main/java/mozilla/lockbox/view/AutofillLockedFragment.kt

```diff
@@ -41,7 +42,7 @@ class AutofillLockedFragment : Fragment(), AutofillLockedView {
             getString(R.string.confirm_pattern)
         )
         try {
-            startActivityForResult(intent, LOCK_REQUEST_CODE)
+            startActivityForResult(intent, Constant.RequestCode.lock)
         } catch (exception: Exception) {
             _unlockConfirmed.onNext(false)
```

Comment:
Replace this with `dispatcher.dispatch(RouteAction.UnlockFallbackDialog)` from the presenter.